lib/utils.ts

```diff
@@ -0,0 +1,17 @@
+import { SALEOR_DOMAIN_HEADER } from "@saleor/app-sdk/const";
+import { FetchAppDetailsDocument } from "../generated/graphql";
+import { getEnvVars } from "./environment";
+import { createClient } from "./graphql";
+import { Request } from "retes";
+
+export async function getAppId(request: Request) {
```

Comment:
Since appId is accessible also from app-bridge I would rename it that it involves the term 'api' to avoid confusion eg `getAppIdFromApi`